src/pages/leftSidebar.js

```diff
@@ -0,0 +1,6 @@
+import { leftSidebarView } from "../views/index.js";
+import { getCategoriesByMovies } from "../data/index.js";
+export const leftSidebar = async (movies) => {
```

Comment:
It is a good practice to add a blank line after your imports (goes for all your files).